src/main/webapp/js/analytics.js

```diff
@@ -12,29 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Sets callbacks for chart instance. */
+/** Sets callbacks for Google Charts instance. */
 google.charts.load('current', {'packages': ['corechart']});
-google.charts.setOnLoadCallback(drawStoresChart);
-
-/** Intializes and draws map onto DOM. */
-function drawStoresChart() {
-  const data = new google.visualization.DataTable();
-  // TODO: Add real data here.
-  data.addColumn('string', 'Store');
-  data.addColumn('number', 'Total');
-  data.addRows([
-    ['Walmart', 10],
-    ['Target', 5],
-    ['Main Street Restaurant', 15],
-    ['Burger King', 7],
-    ['Gap', 13],
-    ['Walgreens', 4],
-    ['Jewel-Osco', 11],
-    ['Shell', 2],
-    ['Starbucks', 12],
-    ['H&M', 20],
-  ]);
+google.charts.setOnLoadCallback(computeStoreChartsAnalytics);
 
+let storeData;
+
+/** Computes and populates DataTable with analytics for stores chart. */
+async function computeStoreChartsAnalytics() {
+  const response = await fetch('/compute-analytics');
+  const analytics = await response.json();
+
+  storeData = new google.visualization.DataTable();
+
+  storeData.addColumn('string', 'Store');
+  storeData.addColumn('number', 'Total');
+
+  for (const [store, total] of Object.entries(analytics)) {
+    storeData.addRow([store, total]);
```

Comment:
See my comment in the other PR, but I'd do the formatting of the store name here.